index.html

```diff
@@ -23,6 +23,7 @@
     }
   </style>
   <link rel="stylesheet" href="css/main.css">
+  <link rel="stylesheet" href="css/search.css">
```

Comment:
Nice you created a style sheet for your search option.

Reply:
Great work adding the css stylesheet to the project